lib/adminui.js

```diff
@@ -38,6 +38,13 @@ function resume(req, res, next) {
     next();
 }
 
+function paramsValidator(req, res, next) {
```

Comment:
this validate all params, which received f.e. from url query or forms, except those that are sent by POST.